File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        left=0
        right=len(numbers)-1
        while left<right:
            current_total = numbers[left]+numbers[right]
            if current_total<target:
                left+=1
            elif current_total>target:
                right-=1
            else:
                return [left+1,right+1]

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.go
func twoSum(numbers []int, target int) []int {
 left,right:=0,len(numbers)-1
 for left<right{
    current_sum:=numbers[left]+numbers[right]
    if current_sum<target{
        left++
    }else if current_sum>target{
        right--
    }else{
        return []int{left+1,right+1}
    }
 }
 return []int{}   
}